modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectEntryValuesException.java

```diff
@@ -76,6 +67,19 @@ public ExceedsMaxFileSize(long maxFileSize, String objectFieldName) {
 
 	}
 
+	public static class ExceedsMaxTextLength
+		extends ObjectEntryValuesException {
+
+		public ExceedsMaxTextLength(int maxLength, String objectFieldName) {
```

Comment:
Since the setting name is max length, I believe the correct order is: "TextMaxLength", not "MaxTextLength".

Reply:
Changed!